test/src/api/commands/web-element/testIsSelected.js

```diff
@@ -0,0 +1,78 @@
+const assert = require('assert');
+const MockServer  = require('../../../../lib/mockserver.js');
+const CommandGlobals = require('../../../../lib/globals/commands-w3c.js');
+const common = require('../../../../common.js');
+const Element = common.require('element/index.js');
+
+describe('element().isSelected() command', function() {
+  before(function (done) {
+    CommandGlobals.beforeEach.call(this, done);
+  });
+
+  after(function (done) {
+    CommandGlobals.afterEach.call(this, done);
+  });
+
+  it('test .element().isSelected() selected', async function() {
+    MockServer.addMock({
+      url: '/session/13521-10219-202/element/0/selected',
+      method: 'GET',
+      response: JSON.stringify({
+        value: true
+      })
+    }, true);
+
+    const resultPromise = this.client.api.element('#signupSection').isSelected();
+    const result = await resultPromise;
+    assert.strictEqual(result, true);
+  });
+
+  it('test .element().isSelected() not selected', async function() {
+    MockServer.addMock({
+      url: '/session/13521-10219-202/element/0/selected',
+      method: 'GET',
+      response: JSON.stringify({
+        value: false
+      })
+    }, true);
+
+    const resultPromise = this.client.api.element('#signupSection').isSelected();
+    const result = await resultPromise;
+    assert.strictEqual(result, false);
+  });
+
+  it('test .element().isSelected() not selected', async function() {
+    MockServer.addMock({
+      url: '/session/13521-10219-202/element/0/selected',
+      method: 'GET',
+      response: JSON.stringify({
+        value: false
+      })
+    }, true);
+
+    const resultPromise = this.client.api.element('#signupSection').isSelected();
+    const result = await resultPromise;
+    assert.strictEqual(result, false);
+  });
+
+  it('test .element().find().isSelected() not selected', async function() {
+    MockServer.addMock({
+      url: '/session/13521-10219-202/element/0/selected',
+      method: 'GET',
+      response: JSON.stringify({
+        value: false
+      })
+    }, true);
+
+    const resultPromise = this.client.api.element('#signupSection').find('#helpBtn').getText();
```

Comment:
You're testing `.getText()` command here?

Reply:
updated in [2cf3ee3](https://github.com/nightwatchjs/nightwatch/pull/4038/commits/2cf3ee382624c7076a9904da3916988ad22c991d). Sorry for this.

Reply:
sorry for missing this. updated.